src/ViewUser.js

```diff
@@ -534,6 +534,12 @@ class ViewUser extends React.Component {
     }
   }
 
+  isLayerOpen = value => {
+    const { layer } = this.props.resources.query;
+
+    return layer && layer === value;
```

Comment:
Is there a reason to check if `layer` exists? If it's `undefined`, then `layer === value` returns `false`